docs/src/content/docs/tr/404.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Not found
```

Comment:
This title should be translated too.

Reply:
Translated in last commit. Also tr locale is defined in astro config file.